change/@fluentui-react-persona-136c281a-d718-41d2-bf77-89ae7c9570b5.json

```diff
@@ -1,5 +1,5 @@
 {
-  "type": "minor",
+  "type": "none",
```

Comment:
This might(?) need to be `prerelease` to get beachball to update the package version.